### ISSUE TITLE
Update to Spring Boot 3

### ISSUE THREAD

**Issue by ffroliva** [citerus/dddsample-core#141]
Is there any thoughts about updating this project to Spring Boot 3? It would be very interesting for the community.

**Comment by c5ms** [comment]
I created the PR:
https://github.com/citerus/dddsample-core/pull/149

**Comment by orende** [comment]
@ffroliva Yes, we tried upgrading earlier this year but had some problems and didn't finish the migration. The PR by c5ms is being reviewed now though, so hopefully this should be done soon.

**Comment by orende** [comment]
This has now been solved by #149, thanks to @c5ms 👍
